babelizer/data/templates/MANIFEST.in

```diff
@@ -1,15 +1,15 @@
-recursive-include {{ cookiecutter.package_name }}/data *
+recursive-include {{ package.name }}/data *
 include LICENSE
 include requirements.txt
 include *.rst
 include *.txt
 include Makefile
 include babel.toml
-recursive-include {{ cookiecutter.package_name }} *.pyx
+recursive-include {{ package.name }} *.pyx
 recursive-include docs *.py
 recursive-include docs *.rst
 recursive-include docs Makefile
```

Comment:
Delete in favor of `build-docs` session

Reply:
I just went ahead and removed the entire manifest file and added a ``tool.setuptools.package-data`` section in *pyproject.toml* for python projects.